README.md

```diff
@@ -0,0 +1,25 @@
+# ORTOA
+
+ORTOA - a One Round Trip Oblivious Access protocol that reads or writes data stored on remote storage *in one round without revealing the type of access*.
```

Comment:
It's no longer an access protocol, but a _family_ of protocols.
![image](https://github.com/ySteinhart1/ORTOA/assets/94937879/0cba84f1-0a28-467f-a8e7-39777892de54)